harness/determined/experimental/_native.py

```diff
@@ -84,11 +86,17 @@ def _submit_experiment(
     auth.initialize_session(master_url, requested_user=None, try_reauth=True)
 
     if test:
+        print(colored("Validating experiment configuration...", "yellow"), end="\r")
```

Comment:
Can you also avoid duplicating this logic both in Native API and CLI? Like putting it in a function somewhere in the `common` package and use it both in Native API and CLI.

Reply:
@shiyuann Since the only repetition is in the 3 spaced out print statements, not entirely sure this is necessary. The experiment creation will differ from native to cli because of the arguments to create_experiment and create_test_experiment so I think migrating this to common may make it more confusing?

Reply:
Ok. This place is fine.